prophetess/templates/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "prophetess.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "prophetess.fullname" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "prophetess.fullname" . }}
+      release: "{{ .Release.Name }}"
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: "RollingUpdate"
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Chart.Name "/templates/configmap.yaml") . | sha256sum }}
```

Comment:
I'm not sure which is correct, or if they both are, but some charts use this as the checksum/config value, while others use `checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}`  (`$.Chart.Name` vs `$.Template.BasePath`)

Reply:
I'm not sure which is better, but the upstream documentation seems to indicate Template.BasePath (https://v2.helm.sh/docs/charts_tips_and_tricks/#automatically-roll-deployments-when-configmaps-or-secrets-change)